critiquebrainz/ws/review/views.py

```diff
@@ -283,10 +283,10 @@ def review_list_handler():
                 "id": "CC BY-NC-SA 3.0",
                 "info_url": "https:\/\/creativecommons.org\/licenses\/by-nc-sa\/3.0\/"
               },
+              "rating": 0,
```

Comment:
Take a look at this:https://github.com/metabrainz/critiquebrainz/pull/125/files#diff-c15d6e193e4bf52d4a77707695d5f205
Rating will be returned in the context of entity rating. So, will have to change that appropriately when we work on ws for rating feature